src/collaboration-platform/GitHubCollaborationPlatform.ts

```diff
@@ -16,9 +16,9 @@ export class GitHubCollaborationPlatform implements CollaborationPlatform {
     this.repoPath = repoPath;
   }
 
-  async getPRForCommit(commit: Commit): Promise<PullRequestInfo | null> {
+  async getPRForCommit(commit: Commit): Promise<PullRequestInfo> {
     // TODO: Implement
-    return null;
+    return {number:1, url:"", title:"", description: "", isOutdated:false}
```

Comment:
This was placed here temporarily to suppress errors. I was having issues will <PullRequestInfo | null> in this function which is why I made the change, don't really see a case where the function would be returning null. But we can probably figure a way out if it is needed.